proto/regen/data/v1alpha2/state.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package regen.data.v1alpha2;
```

Comment:
this isn't being used yet is it? should it go in v1alpha2? or would this be for a v1 release?

Reply:
v1 for release

Reply:
Opened #849 to handle as a follow-up